contracts/src/internal/HyperdriveAdmin.sol

```diff
@@ -30,16 +30,16 @@ abstract contract HyperdriveAdmin is IHyperdriveEvents, HyperdriveBase {
         _checkOptions(_options);
 
         // Ensure that the destination is set to the fee collector.
-        address feeCollector = _feeCollector;
+        address feeCollector = _adminController.feeCollector();
         if (_options.destination != feeCollector) {
             revert IHyperdrive.InvalidFeeDestination();
         }
 
         // Ensure that the caller is authorized to collect fees.
         if (
-            !_pausers[msg.sender] &&
             msg.sender != feeCollector &&
-            msg.sender != _governance
+            msg.sender != _adminController.hyperdriveGovernance() &&
+            !_isPauser(msg.sender)
```

Comment:
i'm sure i'm thinking about this wrong, but it doesn't make sense to me as to why we care who calls `_collectGovernanceFees()`.  as long as the fees go to the feeCollector, we should allow anyone to call it, right?

Reply:
We talked about this offline. I think it's good to have basic admin controls on this function in case the treasury is compromised or there is something sensitive to think about.